### DISCUSSION TITLE
Webhooks in local network not possible

### DISCUSSION THREAD

**Discussion by tomvanderputte** [nocodb/nocodb#7006] [Q&A]
I'm trying to execute a webhook on a DELETE trigger, but I get the following:

DNS lookup [some_ip_address](family:4, host:[some_hostname]) is not allowed. Because, It is private IP address.

I've got a setup of a linux server running a container with NocoDB, and on the same  server I'm running a Flask API on port 5000 (really just a test setup). using either host name or IP adress (10.*.*.* address) will trigger the warning above.

Is there a way to whitelist some ip-addresses or host-names for calling with web-hooks?

**Comment by dstala** [comment]
You will have to set Environment variable `NC_ALLOW_LOCAL_HOOKS=true`
https://docs.nocodb.com/getting-started/self-hosted/environment-variables/
